src/deadline_test_fixtures/deadline/worker.py

```diff
@@ -939,6 +949,12 @@ def configure_agent_user_environment(
             )
 
             cmds.append("systemctl daemon-reload")
+        if config.worker_env_var:
+            for key, value in config.worker_env_var.items():
+                cmds.append(
+                    f'echo "Environment={key}={value}" >> /etc/systemd/system/deadline-worker.service.d/config.conf',
```

Comment:
We should encapsulate the `{key}={value}` bit in quotes as [recommended here](https://man7.org/linux/man-pages/man5/systemd.exec.5.html#ENVIRONMENT) to allow this to handle the case where the value has a space in it. 

I think the following might work (but will need to be tested)
```suggestion
                    f"echo 'Environment=\"{key}={value}\"' >> /etc/systemd/system/deadline-worker.service.d/config.conf",
```